plugins/modules/cloudformation.py

```diff
@@ -75,6 +75,10 @@
       - The path of the CloudFormation stack policy. A policy cannot be removed once placed, but it can be modified.
         for instance, allow all updates U(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/protect-stack-resources.html#d0e9051)
     type: str
+  stack_policy_on_update_body:
+    description:
+      - the body of the cloudformation stack policy only applied during this update.
+    type: str
```

Comment:
```suggestion
  stack_policy_on_update_body:
    description:
      - the body of the cloudformation stack policy only applied during this update.
    type: str
    version_added: 1.3.0
```

When adding a new option we add a `version_added` entry to help folks who might be find the documentation online and be using older versions of Ansible/this collection.